versions/2016-10-31.md

```diff
@@ -319,6 +321,7 @@ Property Name | Type | Description
 Stream | `string` | **Required.** ARN of the Amazon Kinesis stream.
 StartingPosition | `string` | **Required.** One of `TRIM_HORIZON` or `LATEST`.
 BatchSize | `integer` | Maximum number of stream records to process per function invocation.
+Enabled | `bool` | Indicates whether Lambda begins polling the event source.
```

Comment:
This line was added, the rest where style changes to make it compatible with lint